default.nix

```
@@ -130,12 +130,23 @@ let
 
       dalton = callPackage ./pkgs/apps/dalton { };
 
+      dirac = callPackage ./pkgs/apps/dirac rec {
+        stdenv = super.gcc8Stdenv;
+        exatensor = self.exatensor;
+        mpi = super.mpi.override { inherit stdenv; gfortran = super.gfortran8; };
+      };
+
       dkh = callPackage ./pkgs/apps/dkh { };
 
       dftd3 = callPackage ./pkgs/apps/dft-d3 { };
 
       ergoscf = callPackage ./pkgs/apps/ergoscf { };
 
+      exatensor = callPackage ./pkgs/apps/exatensor rec {
```

Jup, it works like this
```suggestion
      exatensor = callPackage ./pkgs/apps/exatensor rec {
        mpi = super.mpi.override { gfortran = super.gfortran8; };
      };
```

---

Let's try that solution first, then.